map/honor-mobile-map-package-expiration-date/README.md

```diff
@@ -0,0 +1,32 @@
+# Honor mobile map package expiration date
+
+Access the expiration information of an expired mobile map package.
+
+![Honor mobile map package expiration date sample](HonorMobileMapPackageExpirationDate.png)
+
+## Use case
+
+The data contained within a mobile map package (MMPK) may only be relevant for a fixed period of time. Using ArcGIS Pro, the author of an MMPK can set an expiration date to ensure the user is aware the data is out of date.
+
+As long as the author of an MMPK has set an expiration date, the expiration date can be read even if the MMPK has not yet expired. For example, developers could also use this API to warn app users that an MMPK may be expiring soon.
```

Comment:
The expiration date can also be read to warn a user that the MMPK may be expiring soon.